code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -264,4 +264,38 @@
 	name = "smashed butter"
 	desc = "Looks like some one has butter fingers."
 	icon = 'icons/effects/tomatodecal.dmi'
-	icon_state = "smashed_butter"
\ No newline at end of file
+	icon_state = "smashed_butter"
+
+/obj/effect/decal/cleanable/virusdish
+	name = "broken virus containment dish"
+	icon = 'icons/obj/virology.dmi'
+	icon_state = "brokendish-outline"
+	density = 0
+	anchored = 1
+	mouse_opacity = 1
+	layer = OBJ_LAYER
+	plane = OBJ_PLANE
+	var/last_openner
+	var/datum/disease2/disease/contained_virus
+
+/obj/effect/decal/cleanable/virusdish/Crossed(var/mob/living/perp)
+	..()
+	FeetStab(perp)
+	infection_attempt(perp,damage = 10,knockdown = 0)
```

Comment:
damage and knockdown are not defined so this will always runtime

Reply:
welp, I gave those args to the wrong proc instead of the one just above